### PR TITLE
Publish on release

### PR DESCRIPTION
**Pull request by sarayourfriend**
Not sure how to test this without merging it!

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      contents: read
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: https://ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```
Comment (AetherUnbound) [inline .github/workflows/publish.yml:23]: Do we need to add this token to the repo?

**Review by AetherUnbound — COMMENTED** [review]
I dig it! I think I'd rather have this triggered on release though, rather than merge to main. Or even releases in addition to merges (that might require another file, since we'll have to get the release version). What do you think? ([spd-lookup](https://github.com/SeattleDSA/spd-lookup/blob/main/.github/workflows/release.yml) does this)

File: .github/workflows/publish.yml (hunk 1 above)
Comment (sarayourfriend) [reply .github/workflows/publish.yml:23]: I _believe_ it's already present by default: https://github.blog/changelog/2021-03-24-packages-container-registry-now-supports-github_token/

**Review by AetherUnbound — APPROVED** [review]
Let's test it out! I'd love to also have it push to a release version tag, but that's something we can add after we verify it works 😄

**Pull request merged**